carrot-log4j-aws-sns/src/main/java/com/carrotgarden/log4j/aws/sns/Appender.java

```diff
@@ -216,14 +225,14 @@ protected boolean ensureCredentials() {
 
             if (url != null) {
                 try {
-                    amazonCredentials = new PropertiesCredentials(url.openStream());
+                    amazonCredentials = new StaticCredentialsProvider(new PropertiesCredentials(url.openStream()));
```

Comment:
I didn't notice that that class was from internal - updating to remove makes sense.  Problem with the classpath provider is that internally it creates a File object with a string path - looks like this appender was written to allow remote file hosting (accessed by url.openStream()). Should we try to continue supporting that or just go with the chain with/out the Classpath provider?